### PR TITLE
Ensure am11/pangyp not fetched via SSH

### PR DESCRIPTION
**Pull request by saper**
Pin am11/pangyp to the particular commit

Should fix the problem mentioned in:
https://github.com/sass/node-sass/commit/579baf322ca48f73b04a009b70220a6c0a3cbc15#commitcomment-11082041

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
     "mkdirp": "^0.5.0",
     "nan": "^1.8.4",
     "npmconf": "^2.1.1",
-    "pangyp": "am11/pangyp",
+    "pangyp": "git://github.com/am11/pangyp.git#f13599f5198b853948429062c2b334010a337342",
```
Comment (xzyfer) [inline package.json]: Should we not just default to the https endpoint to avoid any ssh issues?

**Comment by fourseven** [comment]
I have this problem through the windows meteor build system on v3.0.0:

```
C:\Users\Mathew Hartley>  meteor publish-for-arch fourseven:scss@3.0.0
compileScss: updating npm dependencies -- node-sass, lodash, autoprefixer-core...
npm WARN package.json packages-for-meteor-smartpackage-1qs6hw3@0.0.0 No description
npm WARN package.json packages-for-meteor-smartpackage-1qs6hw3@0.0.0 No repository field.
npm WARN package.json packages-for-meteor-smartpackage-1qs6hw3@0.0.0 No README data
npm ERR! Windows_NT 6.2.9200
npm ERR! argv "C:\\Users\\Mathew
Hartley\\AppData\\Local\\.meteor\\packages\\meteor-tool\\1.1.3\\mt-os.windows.x86_32\\dev_bundle\\bin\\\\node.exe"
"C:\\Users\\Mathew
Hartley\\AppData\\Local\\.meteor\\packages\\meteor-tool\\1.1.3\\mt-os.windows.x86_32\\dev_bundle\\bin\\node_modules\\npm\\bin\\npm-cli.js"
"install"
npm ERR! node v0.10.36
npm ERR! npm  v2.7.3
npm ERR! code ETARGET

npm ERR! notarget No compatible version found:
pangyp@'git+ssh://git@github.com/am11/pangyp.git#f13599f5198b853948429062c2b334010a337342'
npm ERR! notarget Valid install targets:
npm ERR! notarget ["2.0.0","2.0.1","2.0.2","2.1.0"]
npm ERR! notarget
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'node-sass'
npm ERR! notarget


=> Errors while initializing project:

While building plugin `compileScss` in package `fourseven:scss`:
error: couldn't install npm packages from npm-shrinkwrap
```

**Comment by saper** [comment]
That's interesting:

changing the pangyp line to "http" causes the same symptomps as reported by the original poster:
￼￼
https://gist.github.com/941c4f5c16f30d88efb2

https:// seems to work better.

**Comment by xzyfer** [comment]
Unsurprising. Github is https only.

**Comment by am11** [comment]
Thanks @saper! I would say, lets pursue this here: https://github.com/rvagg/pangyp/pull/5.

@rvagg, could you please publish pangyp vNext to npm after the merge? Thanks in anticipation. :gift:

**Comment by xzyfer** [comment]
CI seems pretty unhappy about this.

**Comment by xzyfer** [comment]
This doesn't appear to be a usable solution. We need to wait until it's solved upstream. Addressing this in #942 for now.

**Pull request closed without merging**